<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_model_menu">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: rgb(216, 216, 216)
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">QLabel {
    font-weight: bold;
    font-size:18pt;
	border: 2px solid #606060;
    border-radius: 10px;
    padding: 3px;
	qproperty-alignment: AlignCenter;
}
</string>
         </property>
         <property name="text">
          <string>Modules</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="module_list"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_content">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: rgb(255, 255, 255)
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="6,1,16">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_detail">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_description">
            <property name="styleSheet">
             <string notr="true">QFrame {
	background-color: rgb(216, 216, 216)
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="styleSheet">
                <string notr="true">QLabel {
    font-weight: bold;
    font-size:14pt;
    padding: 3px;
	qproperty-alignment: AlignCenter;
}
</string>
               </property>
               <property name="text">
                <string>Description</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextBrowser" name="text_description"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_control">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame {
	border: 3px;
	background-color: rgb(216, 216, 216);
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="styleSheet">
                <string notr="true">QLabel {
    font-weight: bold;
    font-size:14pt;
    padding: 3px;
	qproperty-alignment: AlignCenter;
}
</string>
               </property>
               <property name="text">
                <string>Options</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="position_mode_frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>AttentionPos: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="position_mode_option"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="layer_mix_frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>LayerMixMode:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="layer_mix_option"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="head_mix_frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>HeadMixMode:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="head_mix_option"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="temperature_frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Temperature:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="temperature_input"/>
                </item>
                <item>
                 <widget class="QPushButton" name="temperature_set_button">
                  <property name="text">
                   <string>Set</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Fontsize:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="fontsize_input"/>
                </item>
                <item>
                 <widget class="QPushButton" name="fontsize_set_button">
                  <property name="text">
                   <string>Set</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_input">
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: rgb(216, 216, 216)
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Input:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_input"/>
          </item>
          <item>
           <widget class="QPushButton" name="confirm_button">
            <property name="text">
             <string>Visualize</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_attention">
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: rgb(216, 216, 216)
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="help_action_en"/>
    <addaction name="help_action_zh"/>
    <addaction name="about_action"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actioncity">
   <property name="text">
    <string>city</string>
   </property>
  </action>
  <action name="help_action_en">
   <property name="text">
    <string>Help(EN)</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="help_action_zh">
   <property name="text">
    <string>Help(ZH)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
